coalib/bearlib/aspects/Smell.py

```diff
@@ -101,13 +101,43 @@ def func(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, z):
         parameters (maybe by creating more functions or using libraries).
         """
 
+
+@MethodSmell.subaspect
+class MethodLength:
+    """
+    This aspect checks for methods and functions with a large number
+    of lines; depending on the the value of the `method_length_count_comment`
+    taste, docs will be counted or not.
+    """
+    class docs:
+        example = """
+        def is_positive(num):
+            if num >= 0:
+                return True
+            else:
+                return False
+        # Here is compact implementation of this method
+
+        def is_positive(num): return True if num >= 0 else False
+        """
+        example_language = 'python'
+        importance_reason = """
+        The longer a method or function is, the harder it becomes to
+        understand and maintain it. Also long methods easily introduce
+        unwanted duplicated code, and bugs to our codebase.
+        """
+        fix_suggestions = """
+        Refactoring methods into smaller more generic methods, making code more
+        compact by using inline and language specific syntax tricks etc.
```

Comment:
Okay so can you add some more suggestions. Probably one or two more. and intead of etc use many methods which enhance/maintain readability and yet reduce clutter.